pandas/tests/test_resample.py

```diff
@@ -696,6 +696,32 @@ def create_index(self, *args, **kwargs):
         factory = self._index_factory()
         return factory(*args, **kwargs)
 
+    _index_fixture_start = datetime(2005, 1, 1)
```

Comment:
as a followup PR. test_resample is getting sufficiently long that it makes sense to split this into some sub-dirs e.g.

``pandas/tests/resample/test_api, test_grouping, test_datetimeindex, test_period`` etc. IOW split out the classes to separate files (to start).